kinetic/distribution.yaml

```diff
@@ -7415,6 +7415,24 @@ repositories:
       url: https://github.com/moose-cpr/moose.git
       version: master
     status: maintained
+  moose-simulator:
```

Comment:
While a dash in the key name isn't strictly forbidden, it is discouraged.  Almost all other packages use an underscore, so please change to that instead.  Thanks.